cadc-download-manager-server/build.gradle

```diff
@@ -27,9 +27,11 @@ dependencies {
     compile 'org.opencadc:cadc-util:1.+'
     compile 'org.opencadc:cadc-log:1.+'
     compile 'org.opencadc:cadc-registry:1.+'
+    compile 'org.opencadc:cadc-rest:[1.2.15,)'
     compile 'org.opencadc:cadc-app-kit:1.+'
```

Comment:
While it's likely that this library won't be used by another client, we have committed to ridding the build files of the `.+` notation.  Please:
 * Replace those with specific minimum versions `[minver,)`
 * Use 1.8 as the `sourceCompatibility` version
 * Use 1.8.4 as the `com.jfrog.bintray` version